packages/connect-core/src/connections/ConnectorInterface.ts

```diff
@@ -2,14 +2,28 @@ import App from '../entities/App'
 import Permission from '../entities/Permission'
 import Repo from '../entities/Repo'
 import Role from '../entities/Role'
+import { AppFilters } from '@aragon/connect-types'
 
 export interface ConnectorInterface {
+  appByAddress(appAddress: string): Promise<App>
```

Comment:
Now that we have the filter. Does the interface need to implement this method?

Reply:
Yes this is still useful to retrieve an app from any organization, while the other one is only for the current organization.

Reply:
Right, thanks for clarifying.